### PR TITLE
Regenerating client library from latest Grafeas v1alpha1 Swagger spec.

### PR DESCRIPTION
**Pull request by vtsao**

### CODE REVIEW

**Comment by clrprod** [comment]
Looks like the tests are failing.

File: v1alpha1/README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # Go API client for grafeas
 
-An API to insert and retrieve annotations on cloud artifacts.
+No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
```
Comment (clrprod) [inline v1alpha1/README.md]: Any way to fix this?

File: v1alpha1/README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
 ## Overview
 This API client was generated by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.  By using the [swagger-spec](https://github.com/swagger-api/swagger-spec) from a remote server, you can easily generate an API client.
 
-- API version: v1alpha1
+- API version: version not set
```
Comment (clrprod) [inline v1alpha1/README.md]: Also any way to fix this?

File: v1alpha1/api/swagger.yaml [hunk 3]
```diff
@@ -52,11 +46,12 @@ paths:
     post:
       tags:
       - "GrafeasProjects"
-      summary: "Creates a new `Project`."
+      summary: "Returns the requested `Occurrence`."
```
Comment (clrprod) [inline v1alpha1/api/swagger.yaml]: Are you sure this is correct?

File: v1alpha1/api/swagger.yaml [hunk 4]
```diff
@@ -86,11 +82,13 @@ paths:
     delete:
       tags:
       - "GrafeasProjects"
-      summary: "Deletes the given `Project` from the system."
+      summary: "Creates a new `Occurrence`. Use this method to create `Occurrences`\n\
```
Comment (clrprod) [inline v1alpha1/api/swagger.yaml]: I'm pretty sure this is wrong?

**Comment by aysylu** [comment]
This looks like an abandoned PR. Closing.

**Pull request closed without merging**